container_linux_test.go

```diff
@@ -175,3 +176,79 @@ func TestShimInCgroup(t *testing.T) {
 
 	<-statusC
 }
+
+func TestContainerHostname(t *testing.T) {
```

Comment:
`hostname` does exist under windows, you may end up with 2 `newlines` on windows though. Stripping the output would prevent having an extra condition though